modules/ocf/templates/environment.prom.erb

```diff
@@ -0,0 +1,8 @@
+<%
+def prometheus_escape(text)
+    return text.gsub(/([\\"\n])/, {"\n"=>"\\n", "\\"=>"\\\\", "\""=>"\\\""})
```

Comment:
The last kv can be written as `'"' => '\n'`, no? Or am I making things up

Reply:
It probably could, but I chose to stick with double quotes everywhere for consistency.

Reply:
It probably could, but I chose to stick with double quotes everywhere for consistency.